### PR TITLE
Test categorical label and position functions

### PR DESCRIPTION
**Pull request by asinghvi17**
at least a little bit

### CODE REVIEW

File: test/conversions.jl [hunk 1]
```diff
@@ -37,3 +37,11 @@ end
         @test fval ≈ sin(val) atol=1f-6
     end
 end
+
+@testset "Categorical values" begin
+    # AbstractPlotting.jl#345
+    a = Any[Int64(1), Int32(1), Int128(2)] # vector of categorical values of different types
+    ilabels = AbstractPlotting.categoric_labels(a)
+    @test ilabels == ["1", "2"]
```
Comment (asinghvi17) [inline test/conversions.jl]:
```suggestion
    @test ilabels == [1, 2]

**Comment by asinghvi17** [comment]
This should be fine, but I'll merge once Gitlab turns green.

**Comment by asinghvi17** [comment]
Need to fix Animations.  Will do tonight.

**Comment by SimonDanisch** [comment]
Jeeez, we need to fix the versions in MakieGallery...cant have the test break all the time, just because a tiny thing changes in a dependency..gonna merge this, since I already started merging other branches with this test failing :D

**Pull request merged**